src/core/LinearExtrudeNode.cc

```diff
@@ -87,9 +87,27 @@ static std::shared_ptr<AbstractNode> builtin_linear_extrude(const ModuleInstanti
     node->filename = filename;
     handle_dep(filename);
   }
-
-  if (parameters["height"].isDefined()) {
-    parameters["height"].getFiniteDouble(node->height);
+  double height = 100.0;
+  Vector3d v(0,0,1);
+
+  if (parameters["v"].isDefined()) {
+    if(!parameters["v"].getVec3(v[0], v[1], v[2], 0.0))
+      LOG(message_group::Error, "v when specified should be  a vector.");
```

Comment:
If the parse failed, we should probably reset the vector to the default (0,0,1). Otherwise, the vector could be partially assigned.

Reply:
An error is created anyway, so it should not matter, but vector is reset now.